code/datums/status_effects/debuffs.dm

```diff
@@ -968,3 +968,26 @@
 /datum/status_effect/cloudstruck/Destroy()
 	. = ..()
 	QDEL_NULL(mob_overlay)
+
+
+//~~~LC13 General Debuffs~~~
+
+/datum/status_effect/rendRedArmor
+	id = "rend red armor"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = 60 //3 seconds
+	alert_type = null
+	var/original_armor
+
+/datum/status_effect/rendRedArmor/on_apply()
+	. = ..()
+	var/mob/living/simple_animal/M = owner
+	original_armor = M.damage_coeff[RED_DAMAGE]
```

Comment:
Not the best way of doing it - if something temporarily debuffs red armor along the way it will basically reset it to some new value and then increase it even higher.
Just add 0.2 on apply and remove 0.2 on remove.

Reply:
Egor the 0.2 add and 0.2 remove is what im doing right now. Thats the issue. 
1 / 1000 = 0.001
0.001 + 0.2 is 0.201
0.201 * 1000 = 201000
So there does need to be a fix. I just have to use divide and multiplication rather than raw adding

Reply:
or maybe i do raw add and subtract it? im not sure what your saying egor.